memcached-image/openSUSE_Tumbleweed.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I really like how you keep all OS distribution specific characteristics separate. For SLES, it should be as simple as replacing this with a SLES version, is that correct ?

Reply:
yes